sites/example-project/src/pages/api/manifest.json/+server.js

```diff
@@ -0,0 +1,85 @@
+import fs from 'fs/promises';
+import path from 'path';
+import preprocess from '@evidence-dev/preprocess';
+import { preprocess as sveltePreprocess } from 'svelte/compiler';
+import { error } from '@sveltejs/kit';
+
+export const prerender = true;
+
+/**
+ * @type {import("@sveltejs/kit").RequestHandler}
+ */
+export async function GET() {
+	// Check that there is somewhere to look for paths
+	// We need to check for src/pages because of example-project
+	const srcPath = path.join(process.cwd(), 'src', 'pages');
+	const pagesPath = path.join(process.cwd(), 'pages');
+
+	const srcExists = await fs
+		.stat(srcPath)
+		.then(() => true)
+		.catch(() => false);
+	const pagesExists = await fs
+		.stat(pagesPath)
+		.then(() => true)
+		.catch(() => false);
+
+	// If we can't find a pages directory, we should abort
+	if (!srcExists && !pagesExists) throw new error(500, './src/pages and ./pages both missing.');
+
+	// trunc to the pages directory
+	const pagesDir = (srcExists ? srcPath : pagesPath).split('pages')[0] + 'pages';
+
+	/**
+	 * @param {string} dirPath
+	 */
+	async function getDirPages(dirPath) {
+		// Get directory contents
+		const dirContent = await fs.readdir(dirPath, { withFileTypes: true });
+		const output = {};
+		if (!dirPath.includes('queries') && !dirContent.some((d) => d.name === 'queries'))
+			return output;
+
+		if (dirContent.some((d) => d.name === '+page.md')) {
```

Comment:
`+page.md` only applies in `example-project`, evidence project pages can be `<anything>.md`

Reply:
This executes in the context of `.evidence/template`, which is after all the pages have been "svelte-kit routified", so `<anything.md>` becomes `<anything>/+page.md`